### ISSUE TITLE
Class 'Composer\Util\Url' not found during project initialization

### ISSUE THREAD

**Issue by peter279k** [php-api-clients/skeleton#28]
As title.

When I try to initialize the new project skeleton, I get this issue.

Here is my setting about this project skeleton:

```
Summary:
--------

 ------------------------------- --------------------- 
  What                            Value                
 ------------------------------- --------------------- 
  Author name                     peter279k            
  Author email                    peter279k@gmail.com  
  Your package                    api-clients/darksky  
  Your project sources location   src/                 
  Your project tests location     tests/               
  Your namespace                  Composer             
  Your test namespace             Composer\Tests       
  Your project namespace          DarkSky              
 ------------------------------- --------------------- 
```

Is the setting wrong?

Please hep me to check that, thanks.

**Comment by WyriHaximus** [comment]
That shouldn't be happening, I'll look into it

**Comment by WyriHaximus** [comment]
Run it with these settings and getting the same error:
```
 ------------------------------- --------------------- 
  What                            Value                
 ------------------------------- --------------------- 
  Author name                     peter279k            
  Author email                    peter279k@gmail.com  
  Your package                    api-clients/darksky  
  Your project sources location   src/                 
  Your project tests location     tests/               
  Your namespace                  ApiClients           
  Your test namespace             ApiClients\Tests     
  Your project namespace          DarkSky              
 ------------------------------- --------------------- 
```

**Comment by peter279k** [comment]
@WyriHaximus, thank you for your concern.

Finally I found another way to do this.

After getting this error exception, the project skeleton has been created.

I change the directory to the current project folder and run the `composer update --no-autoloader --ansi` again.

It's worked successfully.

Is it done for initializing the project skeleton?

Thanks.

**Comment by peter279k** [comment]
After changing into the `"php ~/composer.phar update --no-autoloader --ansi"` and `"php ~/composer.phar update --ansi"` command, it worked successfully without any exceptions.

I don't know the current reason why this issue happens, but I think we should download the `composer` PHAR build firstly before using the `composer create-project` command execution.

**Comment by WyriHaximus** [comment]
@peter279k opened #29 to solve this issue. The reason that `--no-autoloader` line is there is because back in the day it was nesecary to work around an error. It seems fitting that it is causing another one now :joy:

**Comment by WyriHaximus** [comment]
Ok solved the issue in #29
